infra/base-images/base-runner/bad_build_check

```diff
@@ -165,13 +165,32 @@ function check_mixed_sanitizers {
   fi
 }
 
+# Verify that the given fuzz target doesn't crash on the seed corpus.
+function check_seed_corpus {
+  local FUZZER=$1
+  local FUZZER_OUTPUT="/tmp/$(basename $FUZZER).output"
+
+  # Set up common fuzzing arguments, otherwise "run_fuzer" errors out.
+  if [ -z "$FUZZER_ARGS" ]; then
+    export FUZZER_ARGS="-rss_limit_mb=2048 -timeout=25"
+  fi
+
+  bash -c "run_fuzzer $(basename $FUZZER) -runs=0" &> $FUZZER_OUTPUT
+
+  # Don't output anything if fuzz target hasn't crashed.
+  if [ $? -ne 0 ]; then
+    cat $FUZZER_OUTPUT
```

Comment:
can you add an echo line as well before this that fuzz target crashed on execution (with optional seed corpus) or something like that.

Reply:
That's a great idea! Done.